wgpu/examples/bunnymark/main.rs

```diff
@@ -349,3 +349,19 @@ impl framework::Example for Example {
 fn main() {
     framework::run::<Example>("bunnymark");
 }
+
+#[test]
+fn bunnymark() {
+    framework::test::<Example>(
+        concat!(
+            env!("CARGO_MANIFEST_DIR"),
+            "/examples/bunnymark/screenshot.png"
+        ),
+        1024,
+        768,
+        wgpu::Features::default(),
+        framework::test_common::TestParameters::default(),
+        1,
```

Comment:
It's a bit counter-intuitive to provide a bunch of numbers in the parameters. We can live with this, but if you want to consider having another struct with these numbers in the fields, that would be a nice improvement.

Reply:
I agree, especially if we need more tuning in the future.